cluster-autoscaler/cloudprovider/gce/cache.go

```diff
@@ -211,6 +240,7 @@ func (gc *GceCache) InvalidateAllMigInstances() {
 
 	klog.V(5).Infof("Mig instances cache invalidated")
 	gc.instances = make(map[GceRef][]cloudprovider.Instance)
+	gc.instancesRefreshTime = make(map[GceRef]time.Time)
```

Comment:
nit: following the above comment, we should probably rename the variable to like `gc.lastUpdatedTime` / `gc.instancesLastUpdatedTime`, since we're not "refreshing" the cache, but updating the cache.

Reply:
Done